test/spec/EditorCommandHandlers-test.js

```diff
@@ -35,6 +36,8 @@ define(function (require, exports, module) {
         _               = require("thirdparty/lodash");
 
     require("editor/EditorCommandHandlers");
+    
+    var shouldIndentLineComment = Editor.getIndentLineComment();
```

Comment:
The tests should probably run the tests both for `shouldIndentLineComment` `false` and `true`, not by taking the current value from the prefs :fire: